### PR TITLE
Small Next.js app readme updates

### PR DESCRIPTION
**Pull request by amoore108**
Updates the Next.js app readme with a bit more information. Kept it light and directed to the official docs to encourage following idiomatic Next.js as much as possible.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

**Review by alex-magana — APPROVED** [review]

File: ws-nextjs-app/README.md [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
 - Navigate to `http://localhost:7081/kyrgyz/live/cz74kjpyk07t` to see the page
+
+## Running Unit and E2E tests
+
+- Run `yarn test` to run all unit tests
+- Run `yarn test:e2e` to run all e2e tests
+
+## Running Integration tests
+
+- Navigate up to the root of the project (up a level from here)
```
Comment (karinathomasbbc) [inline ws-nextjs-app/README.md]: Consistency with the Storybook section below

```suggestion
- Navigate to the root of the project (up one level from here)
```

File: ws-nextjs-app/README.md [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
+- Run `yarn test:integration --nextJS` to run all integration tests for the Next.js app
+
+## Running Storybook
+
+- Navigate to the root of the project (up a level from here)
```
Comment (karinathomasbbc) [inline ws-nextjs-app/README.md]:
```suggestion
- Navigate to the root of the project (up one level from here)
```

File: ws-nextjs-app/README.md [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+
+## Running Unit and E2E tests
+
+- Run `yarn test` to run all unit tests
+- Run `yarn test:e2e` to run all e2e tests
+
+## Running Integration tests
+
+- Navigate up to the root of the project (up a level from here)
+- Run `yarn test:integration --nextJS` to run all integration tests for the Next.js app
```
Comment (karinathomasbbc) [inline ws-nextjs-app/README.md:27]: Would it help to include the command for updating the snapshots for the NextJS app?

**Review by karinathomasbbc — APPROVED** [review]

File: ws-nextjs-app/README.md (hunk 3 above)
Comment (amoore108) [reply ws-nextjs-app/README.md:27]: Shall add!

**Review by karinathomasbbc — APPROVED** [review]
Thanks for the updates! 🚢

**Pull request merged**